cli/pkg/kctrl/cmd/dev/cmd.go

```diff
@@ -72,10 +72,11 @@ func (o *DevOptions) Run() error {
 	reconciler := cmdlocal.NewReconciler(o.depsFactory, cmdRunner, o.logger)
 
 	reconcileErr := reconciler.Reconcile(configs, cmdlocal.ReconcileOpts{
-		Local:     o.Local,
-		KbldBuild: o.KbldBuild,
-		Delete:    o.Delete,
-		Debug:     o.Debug,
+		Local:           o.Local,
+		KbldBuild:       o.KbldBuild,
+		Delete:          o.Delete,
+		Debug:           o.Debug,
+		DeployResources: true,
```

Comment:
Shouldn't this be false for dev command or am I missing something?

Reply:
The `dev` command will be the only one deploying resources.
Thus, this value is true.

Side-note: This also ensures that we do not have to make changes to any other file affected by the ongoing refactoring 😄 as the value defaults to false for all other commands

Reply:
Never mind, I mixed up things. Just re read the issue and I think this is fine.